### DISCUSSION TITLE
Looking for a community driven GH home for unofficial OpenAPI projects

### DISCUSSION THREAD

**Discussion by displague** [OAI/OpenAPI-Specification#2504] [Q&A]
I'm looking for a home for an OpenAPI project that has many users, stars, and GH forks but has not seen active maintainership recently and has no system of governance.

While the OAS spec advances, many tools that users have come to depend on are left in less than ideal states. There may be competing interests for each of these projects to adapt to the latest standards or continue to support their implementation discrepancies. The user and developer community behind these projects should have some say in this and largely do. Each of these projects takes on or shirks responsibility whenever OAS changes are introduced. A common body of similarly concerned projects could benefit from each other based on similar languages, users, developers, or ideas.

It is understandable that the OAI organization does not want to show favoritism to any single implementation and should want to avoid the situation where an implementation discrepancy with the spec becomes authoritative.

Many of the OAS tools available today are associated with commercial organizations:

* https://github.com/swagger-api (SmartBear)
* https://github.com/mermade

Are these or similar organizations open to adopting existing projects? Would they be able to provide a fair community view that fosters project independence, growth, and health?

Some community-run GH orgs actively maintain community projects or registries:
  * https://github.com/openapitools
 * https://github.com/openapi-contrib
  * https://github.com/apisyouwonthate
  * https://github.com/APIs-guru

These orgs may not be open to adopting competing projects or engineering heavy projects.

1\. Does @OAI have recommendations for projects looking for community governance and maintainership?

2\. Does @OAI have recommendations for community-led initiatives to provide this function?

---

Authors that have expressed interest in having their tools moved to a central org:

* @fotinakis (https://github.com/fotinakis/swagger-blocks)
* @dorthu (https://github.com/Dorthu/openapi3)

Orgs that have expressed interest in hosting disparate projects and hosting community and governance:

* https://github.com/openapi (@sergeysova)

**Comment by kinlane** [comment]
The OAI does not have a formal strategy for tooling, and historically has taken a hands-off stance on the tooling built around specs. I am looking to evolve this stance, and working to develop a way that supports, invests, and helps grow independent projects.

I feel like there should be a commons where we can donate, adopt orphan, and even create new tooling. I would argue that this might now be in the best interest of being a fully endorsed OpenAPI thing (ie. multi-specs, standards, etc.). Happy to discuss and see where this conversation might lead us.

To help drive this conversation I have setup the [API Specification Toolbox](http://api.specificationtoolbox.com/), however I want to build on the work that has come before from Phil's work on his toolbox, Mike's work on various tools, and beyond.

**Comment by handrews** [comment]
I'm going to move this over to the OAI/community repo, which will require converting it to an issue first (discussions aren't enabled on that repo)- apologies for the noise.  I don't know that it will get more attention there, but it is very much a community rather than specification discussion.

**Comment by handrews** [comment]
It turns out you can't migrate a discussion to an issue, only create a new issue, so that issue is OAI/community#15
